har/har_handlers.go

```diff
@@ -67,9 +69,33 @@ func (h *resetHandler) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		log.Errorf("har: method not allowed: %s", req.Method)
 		return
 	}
-	h.logger.Reset()
+
+
+	v, err := parseBoolQueryParam(req.URL.Query(), "return")
+	if err != nil {
+		log.Errorf("har: invalid value for return param: %s", err)
+		rw.WriteHeader(http.StatusBadRequest)
```

Comment:
return here
the way you have it now, `v` will be `false` because that's the zero value of `bool`, and the client will get a StatusNoContent instead of a BadRequest because line 86 will execute.

Reply:
weird, you have a test that covers this and that is proving me wrong..

Reply:
ok I see what's going on.

Still return here.

In the error case, we shouldn't care about `v`. 

What's happening is that since `v` is `true` due to line 98 and lines 81-83 execute. Which means that the logger gets cleared and the log is getting written to rw, even though the client is seeing an error condition - which may be surprising to the client.

Reply:
Yup, `WriteHeaders` guards against multiple invocations, so my failure to return was still passing the test